exec/jdbc/src/main/java/org/apache/drill/jdbc/DrillConnectionConfig.java

```diff
@@ -22,6 +22,24 @@
 
 import net.hydromatic.avatica.ConnectionConfigImpl;
 
+
+// TODO(DRILL-xxxx):  Change public DrillConnectionConfig from class to interface.
```

Comment:
You should probably create a JIRA for this TODO and report it's number here. This way this TODO won't be lost in the sea of already 800+ TODOs